lib/main_common.pm

```diff
@@ -573,4 +574,25 @@ sub load_filesystem_tests() {
     }
 }
 
+sub load_syscontainer_tests() {
+    return unless get_var('SYSCONTAINER_IMAGE_TEST');
+    # pre-conditions for filesystem tests ie. the tests are running based on preinstalled image
+    return if get_var("INSTALLONLY") || get_var("DUALBOOT") || get_var("RESCUECD");
+
+    # setup $serialdev permission and so on
+    loadtest "console/consoletest_setup";
+
+    # Install needed pieces
+    loadtest 'virtualization/libvirtlxc_setup';
+
+    # Register if possible
+    if (get_var("SCC_REGCODE") && get_var("SCC_EMAIL"))
+    {
+        loadtest "jeos/sccreg";
+    }
+
+    # Run the actual test
+    loadtest 'virtualization/syscontainer_image_test';
+}
```

Comment:
The file `lib/main_common.pm` is meant to contain the test plan that is shared between openSUSE and SLE but so fare you only trigger it for SLE. It would be great if you can crosscheck on openSUSE tests as well because we want to support both the same as possible. Any SLE specific code further down (SCC registration, etc.) can be excluded e.g. with

```perl
if (check_var('DISTRI', 'sle')) {
    …
}
```